Lib/collections/__init__.py

```diff
@@ -413,8 +407,8 @@ def _make(cls, iterable):
     _make.__func__.__doc__ = (f'Make a new {typename} object from a sequence '
                               'or iterable')
 
-    def _replace(_self, **kwds):
```

Comment:
Does someone know why the first parameter is called "_self" and not "self".

Reply:
Probably to allow passing `self` as keyword argument. I've seen other code use `__self` similarly.